test/setup.js

```diff
@@ -1,4 +1,6 @@
+/*
 import Vue from 'vue'
 
 Vue.config.productionTip = false
 Vue.config.devtools = false
+*/
```

Comment:
If thsi is commented out. Can it be deleted then?

Reply:
Thanks, done in f5788046bedef4fbdea0987c7234ad93ab153e10